daphne/src/vdaf/mod.rs

```diff
@@ -531,7 +531,13 @@ impl VdafConfig {
         let mut transitions = Vec::with_capacity(num_reports);
         for report_share in agg_job_init_req.report_shares.iter() {
             if processed.contains(&report_share.report_metadata.id) {
-                return Err(DapAbort::UnrecognizedMessage);
+                return Err(DapAbort::UnrecognizedMessage {
+                    detail: format!(
+                        "report ID {} has already been processed",
```

Comment:
This is a little bit ambiguous: Does "processed" mean the report was replayed or already appeared in the current request? I believe the latter is intended.